tests/test-cases/Group6-VIC-Machine/6-04-Create-Basic.robot

```diff
@@ -18,6 +18,51 @@ Resource  ../../resources/Util.robot
 Test Teardown  Run Keyword If Test Failed  Cleanup VIC Appliance On Test Server
 Test Timeout  20 minutes
 
+*** Keywords ***
+Manually Create VCH Folder On VC
+     # Grab the vm folder for the VC
+     ${rc}  ${vm-folder-path}=  Run And Return Rc And Output  govc ls | grep vm
+     Should Be Equal As Integers  ${rc}  0
+     # Create vch named folder for
+     ${rc}=  Run And Return Rc  govc folder.create ${vm-folder-path}/%{VCH-NAME}
+     Should Be Equal As Integers  ${rc}  0
+
+Create Dummy VM In VCH Folder On VC
+    ${vm-folder-path}=  Run  govc ls | grep vm
+    # grab path to the cluster
+    ${rc}  ${compute-path}=  Run And Return Rc And Output  govc ls host | grep %{TEST_RESOURCE}
+    Should Be Equal As Integers  ${rc}  0
+    # Create dummy VM at the correct inventory path.
+    ${rc}  ${output}=  Run And Return Rc And Output  govc vm.create -pool=${compute-path} -net=%{PUBLIC_NETWORK} -folder=${vm-folder-path}/%{VCH-NAME} %{VCH-NAME}
+    Should Be Equal As Integers  ${rc}  0
+
+Create Dummy VM In VCH Folder On ESX
```

Comment:
There is no folder support on standalone ESX, so what's the purpose of this?

Reply:
Correct, but if the folder test is run against an ESXI this function will be run instead of the VC variant. at that will still excercise a code path in the VCH that ensures we error out on esxi when a vm already exists in that path.

Reply:
This keyword is not used. I refactored it out. but forgot to delete it.